static/app/components/tables/simpleTable/index.tsx

```diff
@@ -8,18 +8,17 @@ import Panel from 'sentry/components/panels/panel';
 import {IconArrow} from 'sentry/icons';
 import {space} from 'sentry/styles/space';
 
-interface TableProps {
+interface TableProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
Nit, but we should omit role attribute here since we are overriding it on the table